bin/yaml/c.yaml

```diff
@@ -67,3 +67,23 @@ compilers:
         check_file: chibicc
         targets:
           - main
+    cross:
+      type: s3tarballs
+      arch_prefix:
+      gcc:
+        s3_path_prefix: "{subdir}-gcc-{name}"
+        path_name: "{subdir}/gcc-{name}"
+        untar_dir: "gcc-{name}"
+        check_exe: "bin/{arch_prefix}-gcc --version"
+        arm64morello:
+          - type: tarballs
+            subdir: arm64morello
+            name: morello-10.1-alp1
+            arch_prefix: aarch64-none-elf
+            create_untar_dir: True
+            dir: arm64morello/gcc-aarch64-none-elf-10.1.morello-alp1-x86_64
```

Comment:
If you have any sway with the folks that package these things, the reason for all the `name unused` and random things is that ... there's no consistency! The URL doesn't contain this string (but something _similar_ but capitalized differently), and then this "name" is still needed in a couple places. If it were all consistent we could have one template string and use the `{name}` to control everything...

Anyway, I may still minor refactor this after merging to get rid of _some_ duplicating, but in particular if the URL could differ only in `{name}` that'd be awesome :)

Reply:
I've passed on the feedback to the team that does the packaging :)

Reply:
@mattgodbolt it appears that the recent changes have been made to how the packaging are made.  Things should be more consistent from now on, however the Morello releases are still special, but the releases should be consistent with each other.

Reply:
Awesome stuff. thanks. Is it possible to derive the URL from the version information? That's the key part that's awkward here compared to other installers :)

Reply:
Yes, but I wouldn't do it now.. they'll be a change coming by the end of the year to the content delivery system, this will likely result in different URLs for new packages.  So I'd wait until then.

Reply:
Yes, but I wouldn't do it now.. they'll be a change coming by the end of the year to the content delivery system, this will likely result in different URLs for new packages.  So I'd wait until then.